src/main/com/fulcrologic/rad/database_adapters/sql/migration.clj

```diff
@@ -113,7 +113,9 @@
     (if (= :enum type)
       (let [enums (str/join "," (map #(str "'" % "'") enumerated-values))]
         (if (seq enumerated-values)
-          (format "ALTER TABLE %s ADD COLUMN IF NOT EXISTS %s ENUM(%s);\n" table column enums)
+          (str
```

Comment:
I agree something should be done about this, but this could be a breaking change, so I do not want to do it this way.